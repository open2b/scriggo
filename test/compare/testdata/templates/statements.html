{# render #}

{% var count = 0 %}{% for i := 0; i < 100; i++ %}{% count++ %}{% end for %}
count is {{ count }}
{% if count > 50 %}count is > 50{% else %}count is <= 50{% end if %}
{% var i = interface{}(432) %}
{% switch i.(type) %}{% case string %}
is a string
{% case int %}
is an int
    {% end switch %}
{% i2 := interface{}(-42) %}
{% res := i.(int) - i2.(int) %}
res is {{ res }}
{% for i in []int{1,2,3} %}{{ i }}{% end %}
{% for k in map[int]string{1:"a",2:"b",3:"c"} %}{{ k }}{% end %}
{% show(5) %}
{% show ( i2 ) %}
{% show("scriggo") %}
