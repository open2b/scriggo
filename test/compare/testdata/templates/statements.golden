count is 100
count is > 50


is an int
res is 474
123